manifests/k/Krisp/Krisp/1.48.2/Krisp.Krisp.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Krisp.Krisp
+PackageVersion: 1.48.2
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://cdn.krisp.ai/win/release/v1.48/krisp-v1.48.2-x64.msi
+  InstallerSha256: 425461c812e55c093e89a4ddd9a02f25dd4158fbf4b93c9c98ff566f369a8d5a
+  ProductCode: '{43F5872F-FB6D-4B37-922A-905E3799CB86}'
```

```suggestion
  ProductCode: '{344CF00E-1E0B-47F9-A103-4834FB9C84F8}'
```
fyi: you can get the ProductCode for the MSI installers through `get-wmiobject Win32_Product | Sort-Object Name | Format-Table IdentifyingNumber, Name -AutoSize` in Windows PowerShell